stacks/ollama/devfile.yaml

```diff
@@ -11,55 +11,6 @@ metadata:
     - Starcoder2
   projectType: universal
   language: Polyglot
-  version: 1.0.0
-projects:
-  - name: cde-ollama-continue
-    git:
-      remotes:
-        origin: 'https://github.com/redhat-developer-demos/cde-ollama-continue'
-      checkoutFrom:
-        revision: main
-components:
-- name: udi
-  container:
-    image: quay.io/devfile/universal-developer-image:ubi8-98224a3
-    memoryLimit: 4Gi
-    memoryRequest: 2Gi
-    cpuLimit: 4000m
-    cpuRequest: 1000m
-    mountSources: true
-    sourceMapping: /projects
-- name: ollama
-  attributes:
-    container-overrides:
-      resources:
-        limits:
-          cpu: 4000m
-          memory: 12Gi
-          # nvidia.com/gpu: 1 # Uncomment this if the pod shall be scheduled only on a GPU node
-        requests:
-          cpu: 1000m
-          memory: 8Gi
-          # nvidia.com/gpu: 1 # Uncomment this if the pod shall be scheduled only on a GPU node
-  container:
-    image: docker.io/ollama/ollama:0.1.34
-    mountSources: true
-    sourceMapping: /.ollama
-commands:
-  - id: pullmodel
-    exec:
-      component: ollama
-      commandLine: "ollama pull llama3:8b"
-  - id: pullautocompletemodel
-    exec:
-      component: ollama
-      commandLine: "ollama pull starcoder2:3b"
-  - id: copyconfig
-    exec:
-      component: udi
-      commandLine: "mkdir /home/user/.continue && cp /projects/cde-ollama-continue/continue-config.json /home/user/.continue/config.json"
-events:
-  postStart:
-    - pullmodel
-    - pullautocompletemodel
-    - copyconfig
+  version: 1.0.1
+parent:
+  uri: https://raw.githubusercontent.com/redhat-developer-demos/cde-ollama-continue/main/devfile.yaml
```

Comment:
I think here it would be better to use a specific commit instead of the main branch. Wdyt?

Reply:
This makes sense given that we are versioning the devfile (in this case, 1.0.1). We might want to instead have a branch in the parent devfile repo for each devfile version, and "release" new versions of the devfile, rather than having a rolling release. This is definitely something to gather consensus for https://github.com/devfile/api/issues/1579.

Reply:
Yes +1 I think is important to create some guidelines regarding the versioning of every parent devfile too. This is one of the reasons the spike was opened. For using a release branch I think it also makes sense.

cc @devfile/devfile-services-team in case someone else has input for the usage of a release branch.